src/Microsoft.AspNet.Server.Kestrel/KestrelServerInformation.cs

```diff
@@ -107,5 +110,23 @@ private static bool GetNoDelay(IConfiguration configuration)
 
             return true;
         }
+
+        private static bool GetReuseStreams(IConfiguration configuration)
+        {
+            var reuseStreamsString = configuration["kestrel.reuseStreams"];
+
+            if (string.IsNullOrEmpty(reuseStreamsString))
+            {
+                return false;
+            }
+
+            bool reuseStreams;
+            if (bool.TryParse(reuseStreamsString, out reuseStreams))
```

Comment:
Does `TryParse` throw on `null`? If not you could eliminate the previous `if` block.